src/codemodder/codemods/__init__.py

```diff
@@ -53,10 +53,12 @@ def match_codemods(codemod_include: list, codemod_exclude: list) -> dict:
             name: codemod
             for codemod in DEFAULT_CODEMODS
             if (name := codemod.name()) not in codemod_exclude
+            and (_ := codemod.id()) not in codemod_exclude
```

Comment:
Just curious but is there a reason to write it this way rather than just `codemod.id() not in codemod_exclude`?

Reply:
🤦 of course it should just be `codemod.id() not in codemod_exclude`

If you get a PR in shortly you can go ahead and fix this here and line below